packages/sui-mono/bin/sui-mono-changelog.js

```diff
@@ -56,20 +58,30 @@ function generateChangelog(folder) {
   return new Promise((resolve, reject) => {
     const gitRawCommitsOpts = {path: folder}
     const outputFile = path.join(folder, CHANGELOG_NAME)
-    const output = fs.createWriteStream(path.join(outputFile))
+    const title = '# CHANGELOG'
+    const content = fs.readFileSync(outputFile, 'utf8')
+    const output = fs.createWriteStream(outputFile)
+
+    const name = getWorkspaces().find(path => folder.includes(path))
+    const promise = name ? fetchTags(name) : Promise.resolve()
```

Comment:
To avoid a breaking change we need to keep latests changes at the top. Here would be nice to check if somehow we can use always use streams